rasa/utils/log_utils.py

```diff
@@ -114,7 +115,9 @@ def configure_structlog(
         logger_factory=structlog.stdlib.LoggerFactory(),
         # `wrapper_class` is the bound logger that you get back from
         # get_logger(). This one imitates the API of `logging.Logger`.
-        wrapper_class=structlog.stdlib.BoundLogger,
+        wrapper_class=structlog.make_filtering_bound_logger(log_level),
```

Comment:
To double-check I understand correctly, this will filter out all logs below the indicated level? e.g. If log_level is INFO, then it will filter out DEBUG, but it will process Warning, error and exception levels?

Reply:
If log level is INFO it will filter out all log levels below it (DEBUG, WARNING etc.) and it will not process them (it will be noop - no operation).